frontend/packages/console-shared/src/utils/resource-utils.ts

```diff
@@ -244,6 +245,22 @@ const getAnnotation = (obj: K8sResourceKind, annotation: string): string => {
   return _.get(obj, ['metadata', 'annotations', annotation]);
 };
 
+export const annotationToJSON = (
```

Comment:
We're not converting the annotation *to* JSON. We're converting *from* JSON. So the name is not quite right. I'd call this

```suggestion
export const parseJSONAnnotation = (
```